vms/platformvm/txs/fee/complexity.go

```diff
@@ -72,6 +77,13 @@ const (
 		wrappers.IntLen + // deactivation owner threshold
 		wrappers.IntLen // deactivation owner num addresses
 
+	intrinsicBLSAggregateCompute           = 5     // BLS public key aggregation time is around 5us
+	intrinsicBLSVerifyCompute              = 1_000 // BLS verification time is around 1000us
+	intrinsicBLSPublicKeyValidationCompute = 50    // BLS public key validation time is around 50us
+	intrinsicBLSPoPVerifyCompute           = intrinsicBLSPublicKeyValidationCompute + intrinsicBLSVerifyCompute
+
+	intrinsicWarpDBReads = 3 + 20 // chainID -> subnetID mapping + apply weight diffs + apply pk diffs + diff application reads
```

Comment:
For my own understanding, where does the 3 and 20 come from exactly?

Reply:
- 1 DB read to map the chainID to the subnetID
- 1 DB read to create the weight diff iterator
- 1 DB read to create the pk diff iterator
- 20 DB reads (heuristic) to apply the diffs